XboxKeyboardMouse/XboxController/Emulation/Activate.cs

```diff
@@ -76,38 +74,37 @@ class Activate {
             Program.MainForm.StatusWaiting();
         }
 
-        public static void SendtoController(X360Controller controller) {
-            byte[] report = controller.GetReport();
-            byte[] output = new byte[8];
-
-            // Send our data back to the virtual scp bus.
-            scpbus.Report(CONTROLLER_NUMBER, report, output);
+        public static void ShutDown()
+        {
+            if (shuttingDown == false)
+            {
+                shuttingDown = true;
+                simPad.ShutDown();
+            }
         }
-
-        public static void SendGuide(bool state) {
-            if (state)
-                 Controller.Buttons = Controller.Buttons |  X360Buttons.Guide;
-            else Controller.Buttons = Controller.Buttons & ~X360Buttons.Guide;
-            
-            SendtoController(Controller);
+        
+        public static void SendGuide(bool buttonDown) {
+            if (buttonDown)
+                 state.Buttons |= GamePadControl.Guide;
+            else state.Buttons &= ~GamePadControl.Guide;
+            simPad.Update();
         }
 
         public static void OnProcessExit(object sender, EventArgs e) {
-            scpbus.Unplug(CONTROLLER_NUMBER);
+            ShutDown();
```

Comment:
Would just calling "simPad.ShutDown();" not work here?

Reply:
Honestly I don't think this code is used - Activate doesn't inherit from anything and nothing explicitly adds it as an event.  I'll remove it entirely.